commands/pull_request.go

```diff
@@ -69,6 +69,9 @@ pull-request -i <ISSUE>
 	-l, --labels <LABELS>
 		Add a comma-separated list of labels to this pull request.
 
+	-R, --restrict-maintainer-modify
```

Comment:
I'm still not sold on the flag name `restrict-maintainer-modify`. It doesn't sound like something a human would use to express themselves.

The web UI checkbox is called "allow edits from maintainers". Unchecking that box would be equal to "do not allow edits from maintainers" in human speak. I'm wondering how we could express it in a short flag. Some options:

* `no-edits` (might be too vague/misleading)
* `no-maintainer-edits`
* `restrict-edits`

Any of these make sense? Maybe someone could come up with a better idea?

Reply:
I like the second one, or maybe `no-maintainer-pushes`?

Reply:
I think `no-maintainer-edits` makes the most sense, especially in the context of the existing checkbox saying "allow edits from maintainers".